src/Symfony/Component/Notifier/Bridge/SmsSluzba/composer.json

```diff
@@ -0,0 +1,31 @@
+{
+    "name": "symfony/smssluzba-notifier",
+    "type": "symfony-notifier-bridge",
+    "description": "Symfony sms-sluzba.cz Notifier Bridge",
+    "keywords": ["sms", "sms-sluzba.cz", "notifier"],
+    "homepage": "https://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Dennis Fridrich",
+            "email": "fridrich.dennis@gmail.com"
+        },
+        {
+            "name": "Symfony Community",
+            "homepage": "https://symfony.com/contributors"
+        }
+    ],
+    "require": {
+        "php": ">=8.2",
+        "symfony/http-client": "^7.1",
```

Comment:
We can allow lower version here as well I think

Reply:
Do you mean something like this?
```json
    "require": {
        "php": ">=8.1",
        "symfony/http-client": "^6.4|^7.1",
        "symfony/notifier": "^6.4|^7.1",
        "symfony/serializer": "^6.4|^7.1"
    },
```